src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -60,13 +83,25 @@ public void resetEncoders() {
   }
 
   public void setBrakeMode(){
+    if(brakeMode == false){
```

Comment:
You don’t actually need to know what mode the gearboxes are. You can just blindly set them to coast and brake mode